src/quo2/components/tag.cljs

```diff
@@ -0,0 +1,45 @@
+(ns quo2.components.tag
+  (:require [quo.react-native :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo.theme :as theme]
+            [quo2.components.text :as text]))
+
+(def themes {:light {:background-color colors/neutral-20}
+             :dark {:background-color colors/neutral-80}})
+
+(defn get-value-from-size [size big-option small-option]
+  (if  (= size :big) big-option small-option))
+
+(defn tag-container [size border-color]
+  (merge {:height            (if border-color (get-value-from-size size 34 26) (get-value-from-size size 32 24))
```

Comment:
If components are created from the bottom up, like small/base components first, and then used to create new components with the same padding/margin as Figma, then the height will automatically be 32. And if height is not hard-coded then border-width automatically will be applied outside and we don't need to calculate new height as for `border-width` (as same as you don't need to calculate new width). (no strong opinion, just thinking out loud)

Reply:
yeah so I just added another view around this so I could hardcode the height and with of the child view. Which means the border height will be added outside automatically.